rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -246,77 +247,77 @@ public String defineNodeSource(String sessionId, String nodeSourceName, String i
     }
 
     @Override
     public String editNodeSource(String sessionId, String nodeSourceName, String infrastructureType,
             String[] infrastructureParameters, String[] infrastructureFileParameters, String policyType,
             String[] policyParameters, String[] policyFileParameters, String nodesRecoverable)
             throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.editNodeSource(sessionId,
-                                                                                           nodeSourceName,
-                                                                                           infrastructureType,
-                                                                                           infrastructureParameters,
-                                                                                           infrastructureFileParameters,
-                                                                                           policyType,
-                                                                                           policyParameters,
-                                                                                           policyFileParameters,
-                                                                                           nodesRecoverable));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.editNodeSource(sessionId,
+                                                                                                          nodeSourceName,
+                                                                                                          infrastructureType,
+                                                                                                          infrastructureParameters,
+                                                                                                          infrastructureFileParameters,
+                                                                                                          policyType,
+                                                                                                          policyParameters,
+                                                                                                          policyFileParameters,
+                                                                                                          nodesRecoverable));
     }
 
     @Override
     public String updateDynamicParameters(String sessionId, String nodeSourceName, String infrastructureType,
             String[] infrastructureParameters, String[] infrastructureFileParameters, String policyType,
             String[] policyParameters, String[] policyFileParameters) throws RestServerException, ServiceException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)